Index: test00.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print('test00')
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test00.py b/test00.py
--- a/test00.py	(revision 818b608426cac7018d97a0ea3405e298b118ac6e)
+++ b/test00.py	(date 1717396640455)
@@ -1,1 +1,1 @@
-print('test00')
\ No newline at end of file
+print('test00')
